dashboard/migrations/0001_initial.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
```

Comment:
generated the migration file with `managed=True` for user_selection_table` in case if we want to create the tables using django. So I wasn't sure if this is the directory that should be placed in but that what Django doing it so i followed the approach

Reply:
Yeah I think this is the way we'd like to do it rather than use the script